server/prisma/migrations/20240821214449_classification_update/migration.sql

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
Once add the `category` full text index, then delete all these migrations and create a new single migration with all the changes. (Be sure to delete the four erroneous uuid entries prisma is adding now.)